test/components/DepositWidget.components.test.tsx

```diff
@@ -3,59 +3,111 @@ import { render } from 'enzyme'
+  const onEnableToken = jest.fn<Promise<void>, void[]>()
+  const onSubmitWithdraw = jest.fn<Promise<void>, BN[]>()
+  return (
+    <Row
+      tokenBalances={tokenBalanceDetails}
+      onEnableToken={onEnableToken}
+      onClaim={onClaim}
+      onSubmitDeposit={onSubmitDeposit}
+      onSubmitWithdraw={onSubmitWithdraw}
+    />
+  )
+}
+
+describe('<Row /> not enabled token', () => {
   it('renders single <tr> element', () => {
-    const wrapper = render(<Row tokenBalances={tokenBalanceDetails} />)
+    const wrapper = render(_createRow())
+    wrapper.hasClass
     expect(wrapper.is('tr')).toBe(true)
   })
 
   it('contains 6 <td> elements', () => {
-    const wrapper = render(<Row tokenBalances={tokenBalanceDetails} />)
+    const wrapper = render(_createRow())
     expect(wrapper.find('td')).toHaveLength(6)
   })
 
-  it('contains 2 <button> elements', () => {
-    const wrapper = render(<Row tokenBalances={tokenBalanceDetails} />)
-    expect(wrapper.find('button')).toHaveLength(2)
+  it('contains 1 <button> (enable)', () => {
+    const wrapper = render(_createRow())
+    expect(wrapper.find('button')).toHaveLength(1)
   })
 })
 
-describe('<Row /> with enable', () => {
-  const tokenBalanceDetails: TokenBalanceDetails = {
-    address: '0x0',
-    exchangeBalance: new BN(25),
-    depositingBalance: new BN(0),
-    withdrawingBalance: new BN(11),
-    claimable: true,
-    walletBalance: new BN(2),
+describe('<Row /> enabled token', () => {
+  const tokenBalanceDetails: Partial<TokenBalanceDetails> = {
     enabled: true,
   }
 
   it('renders single <tr> element', () => {
-    const wrapper = render(<Row tokenBalances={tokenBalanceDetails} />)
+    const wrapper = render(_createRow(tokenBalanceDetails))
     expect(wrapper.is('tr')).toBe(true)
   })
 
   it('contains 6 <td> elements', () => {
-    const wrapper = render(<Row tokenBalances={tokenBalanceDetails} />)
+    const wrapper = render(_createRow(tokenBalanceDetails))
     expect(wrapper.find('td')).toHaveLength(6)
   })
 
-  it('contains 3 <button> element', () => {
-    const wrapper = render(<Row tokenBalances={tokenBalanceDetails} />)
+  it('contains 2 <button> elements (deposit and withdraw)', () => {
+    const wrapper = render(_createRow(tokenBalanceDetails))
+    expect(wrapper.find('button')).toHaveLength(2)
+  })
+})
+
+describe('<Row /> claimable token', () => {
+  const tokenBalanceDetails: Partial<TokenBalanceDetails> = {
+    enabled: true,
+    claimable: true,
+    withdrawingBalance: ONE,
+  }
 
+  it('contains 2 <button> elements (claim, deposit, withdraw)', () => {
+    const wrapper = render(_createRow(tokenBalanceDetails))
     expect(wrapper.find('button')).toHaveLength(3)
   })
 })
+
+describe('<Row /> style', () => {
+  it.only('is highlighted', () => {
+    const wrapper = render(
+      _createRow({
+        highlighted: true,
+      }),
+    )
+    expect(wrapper.attr('class')).toMatch(/highlight/)
+  })
+
+  it.only('is enabling', () => {
```

Comment:
BTW, `it.only` skips the other tests in this suite. Any specific reason? Otherwise, just remove the `.only` from them.

With `only`:

PASS  test/components/DepositWidget.components.test.tsx
  <Row /> not enabled token
    ○ skipped renders single <tr> element
    ○ skipped contains 6 <td> elements
    ○ skipped contains 1 <button> (enable)
  <Row /> enabled token
    ○ skipped renders single <tr> element
    ○ skipped contains 6 <td> elements
    ○ skipped contains 2 <button> elements (deposit and withdraw)
  <Row /> claimable token
    ○ skipped contains 2 <button> elements (claim, deposit, withdraw)
  <Row /> style
    ✓ is highlighted (102ms)
    ✓ is enabling (9ms)

Without:

 PASS  test/components/DepositWidget.components.test.tsx
  <Row /> not enabled token
    ✓ renders single <tr> element (113ms)
    ✓ contains 6 <td> elements (16ms)
    ✓ contains 1 <button> (enable) (14ms)
  <Row /> enabled token
    ✓ renders single <tr> element (9ms)
    ✓ contains 6 <td> elements (13ms)
    ✓ contains 2 <button> elements (deposit and withdraw) (5ms)
  <Row /> claimable token
    ✓ contains 2 <button> elements (claim, deposit, withdraw) (9ms)
  <Row /> style
    ✓ is highlighted (5ms)
    ✓ is enabling (4ms)

Reply:
Right! there was a `.only` there...